src/groupeddataframe/grouping.jl

```diff
@@ -520,24 +520,27 @@ a vector or tuple thereof, and `f` must be a callable. If `cols` is a single col
 with a named tuple holding `SubArray` views into these columns.
 
 If the first argument is a vector, tuple or named tuple of such pairs, each pair is
-handled as described above. If a named tuple, field names are used to name
+handled as described above. If it is a named tuple then field names are used to name
 each generated column.
 
 If the first argument is a callable `f`, it is passed a [`SubDataFrame`](@ref) view for each group,
-and the returned `DataFrame` then consists of the returned rows plus the grouping columns.
-If the returned data frame contains columns with the same names as the grouping columns,
+and the returned `GroupedDataFrame` then consists of the returned rows plus the grouping columns.
+If the value returned  by `f` contains columns with the same names as the grouping columns,
```

Comment:
I think this behavior is only supported when an `AbstractDataFrame` is returned, right? So I'd say:
```suggestion
If the data frame returned  by `f` contains columns with the same names as the grouping columns,
```

Reply:
I have said "value" because it does not have to be a data frame. It also can be a named tuple of vectors.

Reply:
OK. Maybe "table" or "table object"? I tend to think of "value" as more or less implying "scalar".

Reply:
fixed